### PR TITLE
create userdata for the instance and open port 80 for http requests

### PR DESCRIPTION
**Pull request by mohanedmoh**

### CODE REVIEW

File: Terraform/AWS/task6/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+env_code     = "Mentorship"
+public_cidr  = ["10.0.1.0/24", "10.0.2.0/24"]
+private_cidr = ["10.0.3.0/24", "10.0.4.0/24"]
+my_ip        = ["90.240.135.206/32"]
```
Comment (geoffrey1330) [inline Terraform/AWS/task6/terraform.tfvars]: add newline at end of file.

File: Terraform/AWS/task6/userdata.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+apt update
+sudo apt install apache2
+service apache2 restart
```
Comment (geoffrey1330) [inline Terraform/AWS/task6/userdata.sh]: add newline at end of file.

File: Terraform/AWS/task6/variables.tf [hunk 3]
```diff
@@ -0,0 +1,6 @@
+variable "env_code" {
+  type = string
+}
+variable "public_cidr" {}
+variable "private_cidr" {}
+variable "my_ip" {}
```
Comment (geoffrey1330) [inline Terraform/AWS/task6/variables.tf]: add newline at end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: Terraform/AWS/task6/variables.tf (hunk 3 above)
Comment (mohanedmoh) [reply Terraform/AWS/task6/variables.tf]: updated

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**